preppal-fe/src/pages/home.tsx

```diff
@@ -1,5 +1,43 @@
-export default function Home() {
-    return (
-        <html>Home</html>
-    )
-}
\ No newline at end of file
+// Import necessary React and TypeScript modules
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import recipeCard from '../components/RecipeCard/RecipeCard';
```

Comment:
small nitpick but I think that these components should be capitalized. 
`RecipeCard`

Reply:
I had it capitalized at first actually, but then I thought that since I'm calling it as a function and not like a react component, it was more appropriate to do it in camelcase.